infra.bs

```diff
@@ -547,6 +548,38 @@ ordered sets; implementations can optimize based on the fact that the order is n
 <a for=list>prepend</a> operation otherwise.
 
 
+<h3 id=tuples>Tuples</h3>
+
+<p>A <dfn export>tuple</dfn> is a specification type consisting of a finite ordered
+immutable-sequence of <dfn for=tuple,pair>items</dfn>, each of which has a
```

Comment:
What is "immutable" about the tuple? The list of names? The values associated with those names?